src/PrestaShopBundle/Twig/Extension/GridExtension.php

```diff
@@ -158,6 +161,36 @@ public function renderColumnHeader(array $column, array $grid)
         ]);
     }
 
+    /**
+     * @param array $grid
+     *
+     * @return bool
+     */
+    public function isOrderingPosition(array $grid)
+    {
+        if (!isset($grid['columns']) || !count($grid['columns'])) {
+            return false;
+        }
+
+        $hasPositionColumn = false;
+        foreach ($grid['columns'] as $column) {
+            if ('position' == $column['type']) {
+                $hasPositionColumn = true;
+                break;
+            }
+        }
+        if (!$hasPositionColumn) {
+            return false;
+        }
+
+        if (isset($grid['sorting']['order_by']) && 'position' == strtolower($grid['sorting']['order_by']) &&
+            isset($grid['sorting']['order_way']) && 'asc' == strtolower($grid['sorting']['order_way'])) {
```

Comment:
having desc is not ordering? the method name is maybe not appropriate, wdyt?

Reply:
more, you can return the result of the condition instead using
```
if (true) {
  return true;
}
return false;
```

Reply:
`isAscendingPosition`, `isAscPosition`, something  like that maybe, it will explain the latest test?

Reply:
I see what you mean, only in this case the name is more centered on the implementation than on the feature, the purpose of this helper is really to detect the "Position drag and drop mode"
For now check sorting parameters is enough, but in product list for exemple there is an orderBy=position_ordering parameter, so the implementation could change in the future depending on use cases